docusaurus/docs/cloud/projects/notifications.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Notifications
+displayed_sidebar: cloudSidebar
+description: View your notifications on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/projects/notifications
+sidebar_position: 3
+---
+
+# Notifications
+
+The **Notification center** could be opened by clicking the bell icon 🔔 in the top navigation of the Cloud dashboard.
```

Comment:
```suggestion
The **Notification center** can be opened by clicking the bell icon 🔔 in the top navigation of the Cloud dashboard.
```
+ PNG/SVG of the bell icon used on the interface, instead of the emoji 😊